aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -123,10 +144,24 @@ export const InPageNavigation = ({ headings }) => {
                         ? undefined
                         : { left: theme.global.borderSize.small }
                     }
+                    direction="row"
+                    align="top"
+                    gap="small"
                   >
                     <Text color="text-strong" size="small" weight="normal">
                       {headingTitle}
                     </Text>
+                    {showUpdate && (
+                      <Box justify="top">
+                        <Text>
+                          <StatusGoodSmall
+                            a11yTitle="section has been updated"
+                            size="10px"
+                            color="#117B82"
```

Comment:
I think we should be able to change this to:

```
<Box background={{dark: true}}>
   <StatusGoodSmall
      a11yTitle="Section has been updated"
      size="10px"
      color="teal"
      height="small"
/>
</Box>

Reply:
Or something like that. The "Box" with dark background will allow teal to render in dark mode. The height on icon will align it with the text line height.